extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -497,5 +498,7 @@ export class AmpStoryPageAttachment extends AMP.BaseElement {
       setTimeout(
           () => toggle(dev().assertElement(this.containerEl_), false), 250);
     });
+
+    setHistoryState(this.win, HistoryState.ATTACHMENT_PAGE_ID, undefined);
```

Comment:
I think the signature function will not accept `undefined`, we can pass `null` instead.

Reply:
Thanks, done : )